src/wizards/Wizard.tsx

```diff
@@ -407,6 +408,8 @@ export class Wizard extends React.PureComponent<WizardProps> {
     }
 
     navigationClick(stepId: number) {
+        const {onNavClick} = this.props;
+        onNavClick && onNavClick(stepId);
```

Comment:
This will call onNavClick() asynchronously. Are we  ok with it ? Also please add a story for same.

Reply:
addressed